### ISSUE TITLE
Basic core structure to be developed allowing future merging of code

### ISSUE THREAD

**Issue by Deepakraj8055** [amosproj/amos2022ws01-firmware-scraper#30]
## User story
1. As a user when i am using the amos firmware scraper
2. I want the code to have a common decided structure that include:

- Main file
- Schedule file
- Directories for the vendor modules
- Logging module

4. So that it allows merging of future codes

## Acceptance criteria
* Core structure matches as per the decided structure and includes the above

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
